_includes/devdoc/guide_dash_features.md

```diff
@@ -110,6 +109,100 @@ had already been confirmed to a block depth of 5 in the blockchain.
 
 {% endautocrossref %}
 
+#### LLMQ InstantSend
+{% include helpers/subhead-links.md %}
+
+{% autocrossref %}
+
+The introduction of Long-Living Masternode Quorums in Dash Core 0.14 provides
+a foundation to further scale InstantSend. LLMQ-based InstantSend removes a
+number of previously required limitations and simplifies the process by decreasing
+the number of P2P messages clients must use.
```

Comment:
in the old system the client still only used one message, `ix` now just `tx` but each `ix` message resulted in each MN in the quorum sending a message around which resulted in a lot of traffic. 
```suggestion
the number of P2P messages sent around the network.
```

Reply:
This sentence was referring to `1 ix + 10*input_count txlvotes` vs `1 tx + 1 islock` (i.e. regular nodes only get an islock and don't have to track all the votes). I'll make some adjustments to mention the network aspect too.